packages/wrangler/src/deployment-bundle/create-worker-upload-form.ts

```diff
@@ -111,6 +111,7 @@ export interface WorkerMetadata {
 	placement?: CfPlacement;
 	tail_consumers?: CfTailConsumer[];
 	limits?: CfUserLimits;
+	annotations?: Record<string, string>;
```

Comment:
Since this is the single field that's not supported in script upload, but supported in version upload, is it worth calling that out in a comment? Is it fairly awkward to split the types into the set of fields supported by either API?